python-avocado.spec

```diff
@@ -820,6 +843,34 @@ Pair-Wise algorithms, also known as Combinatorial Independent Testing.
 %{python3_sitelib}/avocado_framework_plugin_varianter_pict*
 %endif
 
+%package -n python2-%{srcname}-plugins-varianter-cit
+Summary: Varianter with Combinatorial Independent Testing capabilities
```

Comment:
isn't there missing word "plugin"?

Reply:
anyway it's the same as pict so let's get this in and we can always improve it later...